charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -44,8 +44,8 @@ spec:
               name: socket-dir
           resources:
             limits:
-              cpu: 2
```

Comment:
what about set as following, `200m`, `200Mi` could be too small in some load test.
```
              cpu: 1
              memory: 1Gi
```